src/Core/AzureStorageEmulatorAutomation.cs

```diff
@@ -15,6 +15,18 @@ public void Dispose()
             }
         }
 
+        /// <summary>
+        /// Do one-time initialization needed by the emulator.
+        /// Use this before <see cref="Start"/> if you find error "The storage emulator needs to be initialized. Please run the 'init' command."
+        /// </summary>
+        public static void Init()
```

Comment:
What's the rationale for `static` here, as opposed to the previous instance method?

Reply:
It doesn't need access to any other instance member, and it might be used in contexts other than the automation instance.
Mostly I had in mind usage with `[AssemblyInitialize()]`, used with a static method typically separated from other `TestClass`es.
But really I have no strong feelings for this approach, I can change it back :)

Reply:
If you have a use case for `static` I'm OK with leaving it as-is.

Reply:
Actually now that I think of it, it should probably be used right before Start, so I will do change it back.